lib/fog/google/models/compute/server.rb

```diff
@@ -206,6 +206,32 @@ def add_ssh_key username, key
           return self.metadata
         end
 
+        def map_service_accounts(scope_array)
+          gce_scope_aliases = {
+            # list derived from 'gcloud compute instances create --help'
+            'compute-ro' => 'compute.readonly',
+            'compute-rw' => 'compute',
+            'computeaccounts-ro' => 'computeaccounts.readonly',
+            'computeaccounts-rw' => 'computeaccounts',
+            'logging-write' => 'logging.write',
+            'sql' => 'sqlservice',
+            'sql-admin' => 'sqlservice.admin',
+            'storage-full' => 'devstorage.full_control',
+            'storage-ro' => 'devstorage.read_only',
+            'storage-rw' => 'devstorage.read_write'
+              }
+
+          scope_array.map! {
```

Comment:
This is mutating `scope_array`, (`map!` is destructive).  I'd either switch to non-destructive `map` and returning a new array, (I think this is what I'd prefer,) or entitle the method `map_service_accounts!`, and probably call it in the constructor.  Mutating class variables during `save` sounds scary to me.

Stylistically, for a multi-line block, you should use the `do end` syntax, and use brackets for inline.  See [this answer](http://stackoverflow.com/a/5587403/594530).